apps/hash-frontend/src/shared/layout/plain-layout.tsx

```diff
@@ -1,14 +1,18 @@
+import { OwnedById } from "@local/hash-subgraph";
 import { useTheme } from "@mui/material";
 import Head from "next/head";
 import NextNProgress from "nextjs-progressbar";
-import { FunctionComponent, ReactNode } from "react";
+import { FunctionComponent, ReactNode, useContext } from "react";
 
 import { isProduction } from "../../lib/config";
+import { WorkspaceContext } from "../../pages/shared/workspace-context";
 import { CommandBar } from "../command-bar";
 
 export const PlainLayout: FunctionComponent<{
   children?: ReactNode;
 }> = ({ children }) => {
+  const { activeWorkspaceAccountId } = useContext(WorkspaceContext);
```

Comment:
Couldn't this be done in the CommandBar itself? Saves passing it down

Reply:
That makes sense, have moved it in https://github.com/hashintel/hash/pull/2581/commits/3da971ad5b2fe8fee46b9ac3f01bfa3fd0fbc24b